frontend/appflowy_flutter/lib/mobile/presentation/bottom_sheet/show_transition_bottom_sheet.dart

```diff
@@ -0,0 +1,348 @@
+import 'dart:math' as math;
+
+import 'package:appflowy/plugins/base/drag_handler.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:sheet/route.dart';
+import 'package:sheet/sheet.dart';
+
+import 'show_mobile_bottom_sheet.dart';
+
+Future<T?> showTransitionMobileBottomSheet<T>(
+  BuildContext context, {
+  required WidgetBuilder builder,
+  bool useRootNavigator = false,
+  double? elevation,
+  BoxConstraints? constraints,
```

Comment:
Some properties are not being used in this function.

Reply:
Removed!